### ISSUE TITLE
ANSI escape chars shown when using rich.inspect(PrettyDir)

### ISSUE THREAD

**Issue by tddschn** [laike9m/pdir2#66]
The behavior was encountered when using `rich.inspect` on a `PrettyDir` object, although unexpected, might not be a bug.

To reproduce the error, run
```
python3 -c 'from rich import inspect; import pdir; inspect(pdir(print))'
```
Screenshot of output:
<img width="1280" alt="CleanShot-2022-05-29T21-12-46@2x" src="https://user-images.githubusercontent.com/45612704/170870522-68a197ca-6c92-46d0-847c-517168232ed0.png">

Other info
- python: Python 3.10.4 | packaged by conda-forge | (main, Mar 24 2022, 17:43:32) [Clang 12.0.1 ] on darwin
- OS: macOS 12.4
- pdir2: 0.3.5
- [rich](https://github.com/willmcgugan/rich): 12.2.0

**Comment by laike9m** [comment]
pdir's output is supposed to be viewed by human, so I'd say this is expected behavior. Still, could you elaborate a bit why you want to call `rich.inspect` on a `PrettyDir` object?

**Comment by tddschn** [comment]
Sometimes I inspect an object with `rich.inspect` to see the property / attributes (with docstrings) and methods (including signatures and docstrings), since it's faster than finding and reading the docs to get what I want. :)

**Comment by laixintao** [comment]
might be resolved by #67
